### ISSUE TITLE
Support drag and drop

### ISSUE THREAD

**Issue by neurolabusc** [niivue/niivue-ui#22]
@cdrake added support for [drag and drop](https://github.com/niivue/niivue/pull/129). This is great for testing problematic cases. However, it would be great if the niivue-ui could fully support this. Consider the [demo niivue-ui](https://niivue.github.io/niivue-ui/): if you drag and drop a NIfTI image, it is loaded as the background image (good) but the ui decorations list the background image as "mni152.nii". Also, the overlay (hippo.nii) is still loaded, even if it is not registered to the background.

I would suggest the default behavior should be for drag-and-drop to load the image as the background and close overlays. A modifier key might allow the new image to be loaded as a new overlay on top of the currently loaded images.

**Comment by hanayik** [comment]
@neurolabusc, yes I noticed this over the weekend as well. I have some plans to make the UI widgets (names, etc.) more reactive to changes like dropping a new image on the canvas. @cdrake has already included the rxjs library in NiiVue that implements a sort of pub/sub model for reacting to custom events. I'll build on that in order to keep UI state in sync with the NiiVue state.
